unity/Editor/Bindings/MujocoBinaryRetriever.cs

```diff
@@ -55,11 +57,38 @@ public class MujocoBinaryRetriever {
                 Environment.GetFolderPath(Environment.SpecialFolder.UserProfile) +
                 "\\MuJoCo\\bin\\mujoco.dll",
                 mujocoPath + "\\mujoco.dll");
+            GenerateMetaFile(mujocoPath + "\\mujoco.dll");
             AssetDatabase.Refresh();
           }
         }
       }
     }
   }
+  
+  static void GenerateMetaFile(string file) {
+    string metaFileName = file + ".meta";
+
+    // Check if .meta file exists.
+    if (File.Exists(metaFileName)) {
+      Debug.Log("Meta file for " + file + " is already created!");
+      return;
+    }
+
+    // Create a uuid
+    var uuidN = Guid.NewGuid().ToString("N");
+
+    // Write content to .meta
+    using (StreamWriter writer = File.CreateText(metaFileName)) {
+      writer.WriteLine("fileFormatVersion: 2");
+      writer.WriteLine("guid: " + (uuidN.ToString()));
+      writer.WriteLine("DefaultImporter:");
+      writer.WriteLine("  externalObjects: {}");
+      writer.WriteLine("  userData: ");
+      writer.WriteLine("  assetBundleName: ");
+      writer.WriteLine("  assetBundleVariant: ");
+    }
+
+    Debug.Log("Meta file for " + file + " created!");
```

Comment:
How about (or similar): "MuJoCo library imported to location " + filePath

Reply:
OK, I've already updated the output and variable name.